### ISSUE TITLE
An Attention Model for Group-Level Emotion Recognition

### ISSUE THREAD

**Issue by hon9g** [minh364/papers__Today-I-Read#4]
## ABSTRACT
- This paper proposes an approach for improving the classification performance of group-level emotions in images.
![image](https://user-images.githubusercontent.com/26381972/60395646-8b2b6f00-9b71-11e9-800d-874f60729a2f.png)
- There are 2 different and complementary sources of information.
  - **Global Representation** of the entire image.
  - **Local Representation** where only faces in the image.
- While the global representation of the image is learned with a CNN, the local representation is obtained by merging face features through an attention mechanism.
- After two representations learned separately, fused through concatenation in order to obtain the final  classifier.

### RESULTS
They use the approach for `EmotiW 2018 group-level emotion recognition challenge` and ranked 4th with a final accuracy of `64.83%`.

<img src="https://user-images.githubusercontent.com/26381972/60394271-adff5880-9b5c-11e9-9f51-94bc2a6bb01e.png" width="350">
<img src="https://user-images.githubusercontent.com/26381972/60394276-b9528400-9b5c-11e9-8829-6b4f0521f7f5.png" width="350">

### DATASET
- EmotiW 2018 dataset.
  - The images from `Group Affect Database 2.0` 
  - `9815 training data, 4346 validation data, 3011 test data`
  - perceived emtion as `Positive`, `Negative` or `Neutral`.
- [EMOTIC dataset](http://sunai.uoc.edu/emotic/index.html)

### Global-Level CNN
- Learn Global feature from whole image: `DenseNet-161` pre-trained by `ImageNet`
### Local-Level CNN
1. Face Extraction and Alignment: `MTCNN`
2. CNN Model for Face Emotion Recognition: `SphereFace`, a pre-trained CNN model by `CASIA-Webface dataset`. 
### Attention Mechanisms
![image](https://user-images.githubusercontent.com/26381972/60395651-a1392f80-9b71-11e9-8723-1dbcaaaae4f7.png)
